discord_slash/model.py

```diff
@@ -358,3 +360,37 @@ async def wrap():
                     await self._http.delete(self.__interaction_token, self.id)
 
             self._state.loop.create_task(wrap())
+
+
+class PermissionsData:
+    """
+    Slash permissions data object
+
+    :ivar id: Command id, provided by discord.
+    :ivar permissions: List of permissions dict.
+    """
+    def __init__(self, id, permissions, **kwargs):
+        self.id = id
+        self.permissions = permissions
```

Comment:
I think this should create a list of another class, see how `CommandData` does options.

Reply:
7815254